docs/UserGuide.md

```diff
@@ -376,27 +412,35 @@ The command format is provided below:
 
 ![SummaryCommandMonth](images/ug/SummaryCommandMonth.png)
 
-- Provides a financial summary recorded by the user in a month. The month refers to the month that is displayed to the user.
+- Provides a financial summary recorded by the user in a month. The month refers to the month that is displayed to the
+  user.
 - The `MONTH` field is optional, if no month is specified, the application displays the summary for all entries.
 
 ### View entries by category
+
+You've always been a foodie, but how much of your total expenditure is spent on food? You have multiple income streams,
+but which helped you earn the most money? View your entries in a pie chart to find out.
+
 Format: `view t/ENTRY_TYPE`
 
 | Input              | Explanation                                                              |
 |--------------------|--------------------------------------------------------------------------|
 | **`t/ENTRY_TYPE`** | An entry type refers to either `e` for "Expenditure" or `i` for "Income" |
 
 1. View a Pie Chart of all expenditures by categories
-   - Examples: `view t/e`
-   - Expected: `Show graphically all expenditure by category` and a Pie Chart on the right of the application.
+    - Examples: `view t/e`
+    - Expected: `Show graphically all expenditure by category` and a Pie Chart on the right of the application.
 
    ![ViewCommandExpenditureCategory](images/ug/ViewCommandExpenditureCategory.png)
 
 2. View a Pie Chart of all incomes by categories
-   - Examples: `view t/i`
-   - Expected: `Show graphically all income by category` and a Pie Chart on the right of the application
+    - Examples: `view t/i`
+    - Expected: `Show graphically all income by category` and a Pie Chart on the right of the application
 
 ### View entries by month
+
+You went on a shopping spree today. Find out how today's expenditure compared to the rest of the month.
```

Comment:
Should we talk about income too?

Reply:
Have added @EmilyOng, do help me see it is okay :D

Reply:
Looks good!